app/views/profiles/_header.html.haml

```diff
@@ -45,16 +45,18 @@
               .tag.founder
                 = graphical_icon :logo
                 .title Exercism Founder
-
             - else
-              - if user.staff?
-                .tag.staff
-                  = graphical_icon :logo
-                  .title Exercism Staff
-              - if user.maintainer?
-                .tag.maintainer
-                  = graphical_icon :maintaining
-                  .title Maintainer
+              - tags = []
+              - tags << { class: "tag staff", icon: :logo, title: "Exercism Staff" } if user.staff?
+              - tags << { class: "tag maintainer", icon: :maintaining, title: "Maintainer" } if user.maintainer?
+              -# TODO: add user.insider?
+              - tags << { class: "tag maintainer", icon: :insiders, title: "Insider" } if user.maintainer?
```

Comment:
Could you make this a draft until this is done pls? So we don't accidently merge?